app/src/main/res/layout/activity_main.xml

```diff
@@ -3,16 +3,15 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".MainActivity">
```

Comment:
I don't understand why there is MainActivity class, and it's XML file. what is MainActivity supposed to do and how is it relevant to IntroActivity?